### PR TITLE
[TF FE] Review of operations: batch 4

### PR DESCRIPTION
**Pull request by itikhono**
### Details:
Ops in the batch:

- ReduceOps (1 to 1 translators):
   ReduceLogicalOr, ReduceLogicalAnd, ReduceMax, ReduceMean, ReduceMin, ReduceProd, ReduceSum
- ArgMin
- ArgMax
- CropAndResize
- CumSum
- Fill
- IsFinite
- Log1p
- LogSoftmax
- Pack
- RandomUniform
- RandomUniformInt
- Range
- Select
- Size
- Softmax
- TopK
- Unpack
- Where
- ZerosLike

### Tickets:
 - *70338*

### CODE REVIEW

File: ngraph/frontend/tensorflow/src/op/is_finite.cpp [hunk 1]
```diff
@@ -2,6 +2,9 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+// Please review the entire file. These changes have been merged without a detailed review.
+// This comment will be removed after that.
+
```
Comment (sadolini) [inline ngraph/frontend/tensorflow/src/op/is_finite.cpp:7]: line 17 implement TODO or convert to ticket

File: ngraph/frontend/tensorflow/src/op/select.cpp [hunk 2]
```diff
@@ -2,6 +2,9 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+// Please review the entire file. These changes have been merged without a detailed review.
+// This comment will be removed after that.
+
```
Comment (sadolini) [inline ngraph/frontend/tensorflow/src/op/select.cpp:7]: line 19 add node name to Error message to simplify debug and why operation cannot be converted

File: ngraph/frontend/tensorflow/src/op/softmax.cpp [hunk 3]
```diff
@@ -2,6 +2,9 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+// Please review the entire file. These changes have been merged without a detailed review.
+// This comment will be removed after that.
+
```
Comment (sadolini) [inline ngraph/frontend/tensorflow/src/op/softmax.cpp:7]: Implement todo or convert to ticket

File: ngraph/frontend/tensorflow/src/op/log_softmax.cpp [hunk 4]
```diff
@@ -2,6 +2,9 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+// Please review the entire file. These changes have been merged without a detailed review.
+// This comment will be removed after that.
+
```
Comment (sadolini) [inline ngraph/frontend/tensorflow/src/op/log_softmax.cpp:7]: check that input is 2D

**Review by sadolini — COMMENTED** [review]
Overall looks good. I would like to see node names in error messages where possible because it simplifies debug.
